<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VeraCryptPIMDialog</class>
 <widget class="QDialog" name="VeraCryptPIMDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VeraCrypt PIM value</string>
  </property>
  <widget class="QPushButton" name="pbSet">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPIM">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
